test/test_ops.py

```diff
@@ -620,6 +643,10 @@ def test_errors_sparse(self, device, op, layout):
         dtypes=OpDTypes.none,
     )
     @skipIfTorchInductor("Takes too long for inductor")
+    @skipOps(
+        "TestCommon", "test_python_ref_errors", \
+            (('_refs.where', '', 'xpu', None, False),), all_opinfos=python_ref_db
+    )
```

Comment:
Ditto

Reply:
This case passed torch-xpu-ops UT because the NATIVE_DEVICES does not have 'xpu'. When we added xpu in  NATIVE_DEVICES, a different path will be exercised in the test and the case will report a known oneDNN error.